CHANGELOG.md

```diff
@@ -19,6 +19,8 @@
 * [CHANGE] grpcutil.Update: Remove gRPC LB related metadata. #102
 * [CHANGE] concurrency.ForEach: deprecated and reimplemented by new `concurrency.ForEachJob`. #113
 * [CHANGE] ring/client: It's now possible to set different value than `consul` as default KV store. #120
+* [CHANGE] Lifecycler: Default value of lifecycler's `final-sleep` is now `0s` (i.e. no sleep). #121
+* [CHANGE] Lifecycler: It's now possible to change default value of lifecycler's `final-sleep`. #121
```

Comment:
Is this really a breaking change?

Reply:
good question. I don't think both are breaking changes. I'll submit a PR to change this

Reply:
opened https://github.com/grafana/dskit/pull/157 to address that